### DISCUSSION TITLE
[Bug?] Ultima 6 Exult Hot Air Balloon

### DISCUSSION THREAD

**Discussion by Tygaran** [exult/exult#411] [General]
I've managed to build and inflate the ballon, but I can't seem to ride/flt iy.  When I double click the inflated baloon, mt party all gathe on top if it but does not 'enter' it.  Afterwards, any movement on my part just has them all step away from the balloon.

I've also no idea how to pack it back up for inventory.

**Comment by tristantarrant** [comment]
Exult is only for Ultima VII and Serpent Isle. You probably want https://github.com/nuvie/nuvie

**Comment by Tygaran** [comment]
> Exult is only for Ultima VII and Serpent Isle. You probably want https://github.com/nuvie/nuvie

Oh, I know about nuvie, but Ultima 6 Remake for exult was recently release.  Well, it looks like was one previously, but this is a new update.  https://www.ultima6.realmofultima.com/

**Comment by wench** [comment]
The Ultima 6 mod for U7/Exult was not made by us. This isn't really the right place for discussion and help with the mod.
